cmd/skr/README.md

```diff
@@ -19,7 +20,7 @@ The `status` GET method returns the status of the server. The response carries a
 The `attest/raw` POST method expects a JSON of the following format:
 
 ```json
-{	    
+{     
     "runtime_data": "<Base64-encoded blob; the hash digest of the blob will be presented as report data in the raw attestation report>"    

```

Comment:
Since Pawan gave us some customer feedback. We should explain what this runtime-data is for. Basically this runtime_data for proving to a client that the attestation report is for the UVM handling their request, and not one you got last week for a different request on real hardware while you are faking it. They give you some bit random number, you put it in report data and they check the whole signed report contains tha5 same big random number. This is about freshness. https://en.wikipedia.org/wiki/Replay_attack

Reply:
You can do this now. Thanks

Reply:
This is done